AndelaOSP/art-dashboard
# Bg fix unresponsive asset assigning buttons gcp

## What does this PR do?
Fixes bug in staging and production where the asset assigning/unassigning button isn't responding to click events.

## Description of Task to be completed?
The asset assigning/unassigning button isn't responding to click events.
Note: It works on Heroku and on localhost, but not on our staging and prod environments.

## How should this be manually tested?
Once logged into staging or production apps, on the asset lists page, click on any asset. 
On the displayed asset details page description tab, try assigning or un-assigning an asset.
The button should respond to the click by displaying the confirm action modal.

## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/160064437

## Any background context you want to add?
N/a
## Important notes
N/a
## Packages installed
N/a
## Deployment note
N/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
